### PR TITLE
docs: migrating from vuepress to docusaurus

### PR DESCRIPTION
**Pull request by gsi-alejandro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,9 +24,9 @@ That's it, you are ready to use Ottoman.
 
 Supported version are:
 
-| Ottoman     | Nodejs      | Couchbase SDK   | Couchbase Server
-| ----------- | ----------- | --------------- | -----------------
-| ^2.0.0      | ^8.0.0      |  ^3.2.2         | ^6.5.0
+| Ottoman     | Nodejs  | Couchbase SDK | Couchbase Server
+| ----------- |---------|---------------| -----------------
+| ^2.0.0      | ^12.0.0 | ^4.1.3        | ^6.5.0
```
Comment (thejcfactor) [inline README.md:29]: The Couchbase SDK should still be 3.2.2, right?

File: package.json [hunk 2]
```diff
@@ -48,27 +44,21 @@
...
+    "eslint-config-prettier": "8.5.0",
+    "eslint-plugin-prettier": "4.2.1",
+    "husky": "7.0.4",
+    "jest": "27.0.6",
+    "lint-staged": "11.1.2",
+    "prettier": "2.7.1",
+    "pretty-quick": "3.1.1",
+    "rimraf": "3.0.2",
+    "ts-jest": "27.0.4",
+    "typescript": "4.8.4"
```
Comment (thejcfactor) [inline package.json:61]: I think we want to allow for minor version updates, right?

**Review by thejcfactor — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (gsi-alejandro) [reply README.md]: right and fixed

File: package.json (hunk 2 above)
Comment (gsi-alejandro) [reply package.json:61]: no, from previous experience we agreed to remove the ^, installing exactly the package version we are using due to many nodejs packages don't respect the SEMVER, and releasing breaking changes under a minor version.

Comment (thejcfactor) [reply package.json:61]: I think that is fair considering these are devDependencies.  I think others, especially the Couchbase SDK, we should allow for minor bumps.

**Review by thejcfactor — APPROVED** [review]
LGTM 👍

**Pull request merged**